assembly/runtime/collections/index.ts

```diff
@@ -1,15 +1,40 @@
 /**
-* Classes and functions for persistent collections on the blockchain.
-*/
+ * Contract function calls are stateless.
+ *
+ * All contract data is stored in the same key-value data store on the blockchain
+ * (called `Storage` and imported from `near-sdk-as`) with a few convenience
+ * methods for reading, writing, searching and deleting keys and value.
+ *
+ * We also provide a few collections for convenience including
+ * - `PersistentVector`
+ * - `PersistentMap`
+ * - `PersistentDeque`
+ *
+ * These collections wrap the `Storage` class to mimic a Map, Vector (aka. Array) and Deque.
```

Comment:
aka. -> aka 
https://writingexplained.org/aka-or-aka

Reply:
unfortunately they provide zero examples of using `aka` and 3 examples of using `a.k.a.`